clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -588,7 +588,7 @@ private void processDisconnection(List<ClientResponse> responses, String nodeId,
         nodesNeedingApiVersionsFetch.remove(nodeId);
         switch (disconnectState.state()) {
             case AUTHENTICATION_FAILED:
-                log.error("Connection to node {} failed authentication due to {}", nodeId, disconnectState.exception());
+                log.error("Connection to node {} failed authentication due to :", nodeId, disconnectState.exception());
```

Comment:
Is it intentional that there's a space before the colon? Also, is the stacktrace useful here? Or do we just want to print the error message. I haven't looked in detail, so a genuine question.

Reply:
I guess it doesn't matter since this code will be replaced when we integrate the client side part right?

Reply:
@ijuma Thanks for the review. I did think that the stack trace was unnecessary, but left it anyway :-) I have updated it now, but you are right - this code will get updated by the client-side changes.